### DISCUSSION TITLE
希望能有大佬写个鸿蒙NEXT的版本，跪谢

### DISCUSSION THREAD

**Discussion by Timothy-tian** [localsend/localsend#1841] [L-简体中文]
希望能有大佬写个鸿蒙NEXT的版本，跪谢

**Comment by songxing10000** [comment]
同求，现在鸿蒙NEXT上传文件太痛苦了，据说，可以把现有的flutter翻译成鸿蒙啊，最近出的卓易能装不了

**Reply by azhu003** [reply]
> 同求，现在鸿蒙NEXT上传文件太痛苦了，据说，可以把现有的flutter翻译成鸿蒙啊，最近出的卓易能装不了

开源了一个Next版本的LocalSend https://github.com/azhu003/localsend-harmony @songxing10000

**Reply by songxing10000** [reply]
改用自己的签名后，跑在模拟器上，运行报错。

Device info:emulator
Build info:emulator 5.0.0.102(SP1DEVC00E102R4P11log)
Fingerprint:4b82377703f51a1af50ced6d1a835c81d65951eb481d6953cd7c5ea3af67ad59
Module name:com.jerry.localsend
Version:1.0.0
VersionCode:1000000
PreInstalled:No
Foreground:No
Pid:2634
Uid:20020055
Reason:ReferenceError
Error name:ReferenceError
Error message:cannot find record '&rdbplus/src/main/ets/BaseMapper&1.0.5', please check the request path.'/data/storage/el1/bundle/entry/ets/modules.abc'.
Stacktrace:
Cannot get SourceMap info, dump raw stack:
#00 pc 0000000000671da8 /system/lib64/platformsdk/libark_jsruntime.so(623aaba60ba66d828940c18f2d12c7e9)
#01 pc 00000000006722bc /system/lib64/platformsdk/libark_jsruntime.so(623aaba60ba66d828940c18f2d12c7e9)
#02 pc 00000000002c3240 /system/lib64/platformsdk/libark_jsruntime.so(623aaba60ba66d828940c18f2d12c7e9)
#03 pc 000000000016a53c /system/lib64/platformsdk/libark_jsruntime.so(623aaba60ba66d828940c18f2d12c7e9)
#04 pc 0000000000169ee8 /system/lib64/platformsdk/libark_jsruntime.so(623aaba60ba66d828940c18f2d12c7e9)
#05 pc 00000000001efa24 /system/lib64/platformsdk/libark_jsruntime.so(623aaba60ba66d828940c18f2d12c7e9)
#06 pc 00000000005669f0 /system/lib64/platformsdk/libark_jsrun
> > 同求，现在鸿蒙NEXT上传文件太痛苦了，据说，可以把现有的flutter翻译成鸿蒙啊，最近出的卓易能装不了
> 
> 
> 
> 开源了一个Next版本的LocalSend https://github.com/azhu003/localsend-harmony @songxing10000
